zanata-rest-client/src/main/java/org/zanata/rest/client/ClientUtil.java

```diff
@@ -107,4 +110,25 @@ public static String getBaseURL(String movedTo) {
             return movedTo;
         }
     }
+
+    /**
+     * Extract filename from response header.
+     *
+     * e.g. Content-Disposition=[attachment; filename="RPM.po"]
+     *
+     * @param headers
+     * @return
+     */
+    public static String getFileNameFromHeader(MultivaluedMap<String, String> headers) {
+        final String CONTENT_DISPOSITION_HEADER = "Content-Disposition";
+        final String FILENAME_PATTERN = "filename=\"(.*?)\"";
```

Comment:
- [ ] Can the filename ever contain a double-quote character? If so this will fail for an escaped quote character because it uses lazy repeat (`*?`). Please figure out whether there can be a quote, and if so how it can be escaped. I can help make a regex that will match it accurately.